### PR TITLE
Refactor gphdfs test cases to work with pg_regress

### PR DESCRIPTION
**Pull request by 50wu**
to verify all tests can run successfully on old Hadoop Distribution,take HDP 2.3.2 for example.

### CODE REVIEW

**Comment by yydzero** [comment]
for build_test_jar.sh, do we need to put this file into repo? looks like it has many hardcoded path

**Comment by yydzero** [comment]
for gphdfs_regress_schedule, is there any chance to make them run in parallel?

**Comment by yydzero** [comment]
for boolean_ExtwriteToHdfsToExtread.source, there are many INFO messages, is there anyway to adjust error level to skip INFO?

**Comment by 50wu** [comment]
for build_test_jar.sh, it will generate the required jar file for the test, already replace the hard-coded path.
for gphdfs_regress_schedule, because there will be an impact between each case(the file path is  same),so temporarily can not run in parallel.
for boolean_ExtwriteToHdfsToExtread.source, other files also have INFO messages before the migration from tinc and I wrote init_file to ignore these INFO messages.

File: gpAux/extensions/gphdfs/regression/output/rescan.source [hunk 1]
```diff
@@ -48,133 +46,129 @@ O_ORDERPRIORITY  CHAR(15) ,
...
-         38 |    124828 | O             |     82500.05 | 1996-08-21  | 4-NOT SPECIFIED | Clerk#000000604 |              0 | haggle blithely. furiously express ideas haggle blithely furiously regular re
-         35 |    127588 | O             |    253724.56 | 1995-10-23  | 4-NOT SPECIFIED | Clerk#000000259 |              0 | zzle. carefully enticing deposits nag furio
-         66 |    129200 | F             |    103740.67 | 1994-01-20  | 5-LOW           | Clerk#000000743 |              0 | y pending requests integrate
-         32 |    130057 | O             |    208660.75 | 1995-07-16  | 2-HIGH          | Clerk#000000616 |              0 | ise blithely bold, regular requests. quickly unusual dep
-          4 |    136777 | O             |     32151.78 | 1995-10-11  | 5-LOW           | Clerk#000000124 |              0 | sits. slyly regular warthogs cajole. regular, regular theodolites acro
+         71 |      3373 | O             |    276992.74 | 01-24-1998  | 4-NOT SPECIFIED | Clerk#000000271 |              0 |  express deposits along the blithely regul
```
Comment (50wu) [inline gpAux/extensions/gphdfs/regression/output/rescan.source]: tried en_US.UTF-8 and zh_CN.UTF-8, all passed. will investigate this later. (created a chore)

**Pull request merged**